### PR TITLE
update sequel.html.md to fix 404

### PR DESCRIPTION
**Pull request by aksie**
fxied link to https://github.com/jeremyevans/sequel

### CODE REVIEW

File: source/ruby/integrations/sequel.html.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Read more about [`:instrument_sequel`][instrument-sequel].
 
 ### sequel-rails integration
 
-When using the [sequel-rails](sequel-rails) gem the instrumentation from the
+When using the [sequel-rails](https://github.com/jeremyevans/sequel) gem the instrumentation from the
```
Comment (tombruijn) [inline source/ruby/integrations/sequel.html.md]:
```suggestion
When using the [sequel-rails][sequel-rails] gem the instrumentation from the
```

This uses the existing link and points to the correct project

Comment (aksie) [reply source/ruby/integrations/sequel.html.md]: Hi, 
Robert told me it should link to https://github.com/jeremyevans/sequel

But I guess the point is it should not have been a link? / what do the square brackets do?

Comment (tombruijn) [reply source/ruby/integrations/sequel.html.md]: Sequel rails is another project than sequel itself. The square brackets reference the list of urls at the bottom of the file

**Comment by matsimitsu** [comment]
So we've gone from round brackets to square brackets, correct?

**Comment by aksie** [comment]
yes.
this wasn't meant to link to a page but to the list of urls in the footer.

**Pull request merged**